brave/browser-laptop
# Make welcome screen show up on first time run

Submitter Checklist:


Auditors: @bsclifton
Close #9423

Test Plan:
```
npm run test -- --grep="aboutWelcomeReducer"
npm run unittest -- --grep="tabs API unit tests create"
npm run unittest -- --grep="sessionStore unit tests defaultAppState"
```

Manual test plan:
1. Have your session as-is
2. Open Brave with this PR
3. Open a new tab
4. About welcome shouldn't show
5. Clear session
6. Open Brave
7. New tab should show, followed by about:welcome as the second tab, active
8. Open another tab by any means
9. about:newtab should show alone

Reviewer Checklist:

Tests
